### PR TITLE
Add beacon network

### PR DESCRIPTION
**Pull request by gsfk**
Add handling for beacon network. Handled as extra beacon configuration, no new containers. Accompanies [beacon pr #93.](https://github.com/bento-platform/bento_beacon/pull/93), [bento_public pr #165](https://github.com/bento-platform/bento_public/pull/165).

TODO:
- [x]  add network config file
- [x] update bentoctl beacon methods
- [x] add feature switch for network
- [x] update docs

### CODE REVIEW

File: etc/bento_deploy.env [hunk 1]
```diff
@@ -10,6 +10,7 @@ BENTO_GATEWAY_USE_TLS='true'
 
 BENTO_BEACON_ENABLED='false'  # Set to true if using Beacon!
 BENTO_BEACON_UI_ENABLED='false'
+BENTO_BEACON_NETWORK_ENABLED='false'
```
Comment (davidlougheed) [inline etc/bento_deploy.env:13]: add this variable to `default_config.env` and `bento_dev.env` too

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: docs/installation.md [hunk 2]
```diff
@@ -158,6 +158,14 @@ If using Beacon, first copy the configuration file:
 Then update any config values as needed at `lib/beacon/config/beacon_config.json` 
 and `lib/beacon/config/beacon_cohort.json`.
 
+If using the beacon network, copy the configuration file:
```
Comment (davidlougheed) [inline docs/installation.md:161]: capital B for beacon for consistency

File: docs/installation.md [hunk 3]
```diff
@@ -158,6 +158,14 @@ If using Beacon, first copy the configuration file:
...
+
+```bash
+./bentoctl.bash init-config beacon-network
+```
+
+and update values at `lib/beacon/config/beacon_network_config.json`.
+
```
Comment (davidlougheed) [inline docs/installation.md:168]: Can a note about this be added to the v17 migration guide? Similar to here, basically

**Review by davidlougheed — CHANGES_REQUESTED** [review]
minor: a documentation request; otherwise lgtm

File: docs/installation.md (hunk 2 above)
Comment (gsfk) [reply docs/installation.md]: (╯°□°）╯︵ ┻━┻

File: docs/migrating_to_17.md [hunk 4]
```diff
@@ -84,6 +84,21 @@ bento_authz public-data-access counts
 ```
 
 
+## 5. Optionally, add Beacon network
+
+To host a network of beacons, with a corresponding UI in bento_public, first copy the config file:
```
Comment (davidlougheed) [inline docs/migrating_to_17.md:89]: either Bento Public or `bento_public` with code ticks around it

File: docs/migrating_to_17.md [hunk 5]
```diff
@@ -84,6 +84,21 @@ bento_authz public-data-access counts
...
+```
+
+then update values at `lib/beacon/config/beacon_network_config.json`. Activate the network by adding (or modifying) this value in local.env:
+
+
+```bash
+BENTO_BEACON_NETWORK_ENABLED='true'
+```
+
 ## 5. Start Bento
```
Comment (davidlougheed) [inline docs/migrating_to_17.md:102]: make this 6

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: docs/migrating_to_17.md (hunk 5 above)
Comment (gsfk) [reply docs/migrating_to_17.md:102]: They're so closely tied I didn't see the point

Comment (davidlougheed) [reply docs/migrating_to_17.md:102]: even still, I don't think it makes sense to have two step 5s, with one of them being optional and the other being required (and coming after the first step 5)

Comment (gsfk) [reply docs/migrating_to_17.md:102]: right, I was thinking of the env step as optional also, but I guess it needs to be throw in regardless

**Review by davidlougheed — APPROVED** [review]
lgtm

**Pull request merged**